src/enkf/readconvobs.f90

```diff
@@ -324,12 +326,15 @@ subroutine get_num_convobs_nc(obspath,datestring,num_obs_tot,num_obs_totdiag,id)
         endif
         if (obtype == '  q') then
            allocate(Forecast_Saturation_Spec_Hum(nobs_curr))
-           call nc_diag_read_get_var(iunit, 'Forecast_Saturation_Spec_Hum', Forecast_Saturation_Spec_Hum)
+           if ( qobs_pseudo_rh ) then
+              call nc_diag_read_get_var(iunit, 'Forecast_Saturation_Spec_Hum', Forecast_Saturation_Spec_Hum)
+           else
+               Forecast_Saturation_Spec_Hum=1.
```

Comment:
Could you elaborate more on why set Forecast_Saturation_Spec_Hum=1?  Is it specific humidity or relative humidity?

Reply:
At the moment, q obs are assimilated as q_obs/qsat_mod. I put this code in to test assimilating the q obs as q_obs (since qsat is used in multiple places to scale the obs, obs error, etc, setting it to 1 was my lazy way to achieve this). However, I've been looking into this since I submitted the PR, and I no longer think that using q_obs without scaling by qsat is a good idea (longer term, I'll test using q_obs/qsat_obs). I was planning to remove this from a future PR, but will do it now since you've flagged it. I'm about to go to an appointment, but will push an update later today with this removed.